### PR TITLE
Add mediafile AU

### PR DESCRIPTION
**Pull request by tsiegleauq**
browsing example data works
Selecting mediafiles in a search value selector was not covered here

### CODE REVIEW

File: client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts [hunk 1]
```diff
@@ -216,18 +218,24 @@ export class MediafileListComponent extends BaseListViewComponent<ViewMediafile>
...
     public ngOnInit(): void {
+        super.ngOnInit();
         super.setTitle('Files');
 
         this.repo.getDirectoryIdByPath(this.route.snapshot.url.map(x => x.path)).then(directoryId => {
             this.changeDirectory(directoryId);
         });
 
         // Observe the logo actions
+        /**
```
Comment (FinnStutzenstein) [inline client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts:229]: The logo/font handling has changed completly: It should also be done in this PR.

Mediafiles have template fields `used_as_logo_$_in_meeting` and `used_as_font_$_in_meeting`. The values to replace are the places for the mediafiles. E.g. for logos there can be `web_header`, `pdf_header`, `pdf_footer`, ... (see https://github.com/OpenSlides/OpenSlides/blob/master/openslides/core/config_variables.py#L255 without the `logo_` prefix)

This list must now be defined in the client - the server does not know about these fields anymore. It implies, that the client can define fields, as it likes without server changes.

So, if a mediafile is used as the `web_header` and `pdf_header`, the fields `used_as_logo_web_header_in_meeting` and `used_as_logo_pdf_header_in_meeting` are set.

Maybe the `MediaManageService` can save the list of possible places and do some handling of that.

File: client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.html [hunk 2]
```diff
@@ -41,7 +41,7 @@ <h2>{{ 'Files' | translate }}</h2>
                     *ngIf="!last"
                 >
                     <span class="folder-text">
-                        {{ directory.filename }}
+                        {{ directory.name }}
```
Comment (FinnStutzenstein) [inline client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.html:44]: please change to title

File: client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts (hunk 1 above)
Comment (FinnStutzenstein) [reply client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts:229]: For selecting a place in the dot-menu: There will be a dedicated action for setting a mediafile in a meeting as X (e.g. `projector_header`).

Comment (tsiegleauq) [reply client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts:229]: It's still not clear how and from where to get the information about the currently used logo(s) and fonts, this this feature is way beyond the scope of this pull request.

**Pull request merged**